Example/GiniVision/ScreenAPIViewController.swift

```diff
@@ -84,13 +84,13 @@ class ScreenAPIViewController: UIViewController {
         let vc = GINIVision.viewController(withDelegate: self, withConfiguration: giniConfiguration)
         
         // 3. Present the Gini Vision Library Screen API modally
-        presentViewController(vc, animated: true, completion: nil)
+        self.present(vc, animated: true, completion: nil)
```

Comment:
Maybe it's just an Objective-C habit, but I prefer seeing the `self.` so I know it's a member of the same class. Anyway, I've omitted it in this case.